### ISSUE TITLE
Export and Import of Hotkeys

### ISSUE THREAD

**Issue by kayrie** [rollerderby/scoreboard#645]
For travelling SOs it would be great, if hotkey could be imported and exported.

As there are many different keyboard layouts in most parts of the world, it would be beneficial to store keyboard layout used to set the hotkey and to ask, if the hotkeys should be imported when the importing device has a different layout.

**Comment by bullseye555** [comment]
@kayrie - Download of hotkeys has always been included as part of _Download All Data_ and since [5.0.11](https://github.com/rollerderby/scoreboard/releases/tag/v5.0.11) has had it's own section in the _Games / Teams / Rulesets_ page so that you can download individual settings more simply
![image](https://github.com/rollerderby/scoreboard/assets/28804763/18608982-2af5-48ab-8f5a-6dce0aa551e1)

To import, you then just select the file and choose the _Import JSON_ option

Regarding the additional ask of storing the Keyboard Layout - I'll let @frank-weinberg comment on that as I don't know the difficulty / practicality of it - my biggest concern would be variations in "language" even in the same country. EG, here in Australia the keyboard layout could be US, UK, or AU - with the only real difference between the 3 being that the $ on the AU & US layout being replaced with £. I'm not sure I'd want to be having a warning message thrown at me saying the keyboard layout is different on the new device, when in substance it's not (it would just get confusing to operators who aren't as well versed in these sorts of things

**Comment by frank-weinberg** [comment]
I'm not sure there even is a way to detect the keyboard layout from the browser. And if there is, if there is a way of storing it that is consistent across operating systems. Then there is the problem Bullseye points out that there may be subtle differences in layout that may or may not be relevant for the hot keys used. And the import interface currently has no way of submitting extra information in addition to the file. And adding a possibility to ask the user questions about the import would be even more complicated.

So in summary this feature would require a very significant amount of effort that I don't see justified by how useful it is.
